### PR TITLE
Feature/instructor styling

### PR DESCRIPTION
**Pull request by JesusNavarrete215**
https://www.loom.com/share/ad44a31c3016478ab148c21519e43f82
## Description

I reflected the styling for the parent calendar event card on the instructor calendar. I included a few changes to the color and radius of the ant design. I also changed the header from a fixed position to a sticky position so it wouldnt be overlapping with the body content. 

Fixes # (issue)

## Loom Video

Please paste a link here of a quick loom video walking through what you did in your PR .

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [ ] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/ParentBooking/ParentBookingCard.js [hunk 1]
```diff
@@ -2,6 +2,23 @@ import React from 'react';
...
+//     subject: 'Flintstone'
+
+//   })
+//   .then(function (response) {
+//     console.log(response);
+//   })
+//   .catch(function (error) {
+//     console.log(error);
+//   });
+// }
```
Comment (RoseGabriely) [inline src/components/pages/ParentBooking/ParentBookingCard.js:21]: Does this commented-out code need to be merged into our main codebase? Maybe there's a decision to make about whether you want to finish implementing whatever was started here or get rid of it. We also want to consider the advantages and disadvantages of making an API call locally vs adding it to our global store and pulling from there.

File: src/index.js [hunk 2]
```diff
@@ -67,7 +67,7 @@ function App() {
 
   return (
     <Security {...config} onAuthRequired={authHandler}>
-      <NavBar />
+      {/* <NavBar /> */}
```
Comment (RoseGabriely) [inline src/index.js:70]: Commented out code!

File: src/components/pages/ParentBooking/ParentBookingContainer.js [hunk 3]
```diff
@@ -62,8 +62,8 @@ const ParentBookingContainer = props => {
               </Card.Grid>
             </TabPane>
           ))}
-          <TabPane tab="1:1 Sessions" key="3">
-            1:1 Sessions
+          <TabPane tab="My Courses" key="3">
+            My Courses
```
Comment (RoseGabriely) [inline src/components/pages/ParentBooking/ParentBookingContainer.js:66]: Thanks for this change! Are we populating this tab(and other tabs) with our endpoint data or will that be a separate PR?

File: src/styles/ParentStyles/index.less [hunk 4]
```diff
@@ -110,15 +110,18 @@ li {
...
-  background-color: #daf4ff;
-  border-radius: 25%;
-  overflow: unset;
-  margin-bottom: 0;
+    background-color: #daf4ff;
+    border-radius: 25%;
+    overflow: unset;
+    margin-bottom: 0;
+    
+
```
Comment (RoseGabriely) [inline src/styles/ParentStyles/index.less:124]: Couple of extra empty lines to delete here!

File: src/styles/ParentStyles/index.less [hunk 5]
```diff
@@ -110,15 +110,18 @@ li {
   text-align: center;
   width: 100%;
 }
+
```
Comment (RoseGabriely) [inline src/styles/ParentStyles/index.less:113]: Let's keep this consistent with the rest of the styling in this file, it looks like it's been decided to not add spaces between the selectors. If you're trying to section out a space in this file for a particular set of styles, use comments to express that and create the spacing.

File: src/styles/global.less [hunk 6]
```diff
@@ -56,7 +56,7 @@
     align-items: center;
     justify-content: space-evenly;
     margin-bottom: 0%;
-    position: fixed;
+    position: sticky;
     z-index: 2;
 }
```
Comment (RoseGabriely) [inline src/styles/global.less:62]: Mind removing this space?

File: src/styles/header.less [hunk 7]
```diff
@@ -55,14 +55,6 @@ width: 3em;
 height: 3em;
 }
```
Comment (RoseGabriely) [inline src/styles/header.less:57]: Mind removing this space?

File: src/styles/header.less [hunk 8]
```diff
@@ -73,10 +65,6 @@ height: 3em;
     height: 2.4rem;
     border: transparent;
     border-radius: 4px;
-    // margin-block-start: 1.33em;
-    // margin-block-end: 1.33em;
-    // margin-inline-start: 0px;
-    // margin-inline-end: 0px;
```
Comment (RoseGabriely) [inline src/styles/header.less:68]: Mind removing this space?

**Review by RoseGabriely — CHANGES_REQUESTED** [review]
Overall great work, just some minor clean-up to do, and a decision to make about the state you began implementing. We should have all of the endpoints you'd need to implement state if you choose to do so, feel free to reach out if you need help finding them. It would be awesome to ship this feature fully built out and connected to the backend!

File: src/components/pages/ParentBooking/ParentBookingCard.js (hunk 1 above)
Comment (JesusNavarrete215) [reply src/components/pages/ParentBooking/ParentBookingCard.js:21]: Great catch! Since Elchin and I are working on the same task, I placed the commented code where we might set up the api post to add a course to the students "My Courses". I was also going back and forth on our branch and on my stash.

File: src/index.js (hunk 2 above)
Comment (JesusNavarrete215) [reply src/index.js:70]: The Nav was getting in the way so we decided to comment it out. Now, we should have the styling for the global NavBar done.

File: src/styles/ParentStyles/index.less (hunk 4 above)
Comment (JesusNavarrete215) [reply src/styles/ParentStyles/index.less:124]: I will remove the extra spacing, thanks

File: src/styles/ParentStyles/index.less (hunk 5 above)
Comment (JesusNavarrete215) [reply src/styles/ParentStyles/index.less:113]: I will remove the spacing.

File: src/styles/global.less (hunk 6 above)
Comment (JesusNavarrete215) [reply src/styles/global.less:62]: I will remove the spacing.

File: src/styles/header.less (hunk 7 above)
Comment (JesusNavarrete215) [reply src/styles/header.less:57]: I will remove the spacing.

File: src/styles/header.less (hunk 8 above)
Comment (JesusNavarrete215) [reply src/styles/header.less:68]: I will remove the spacing.

File: src/components/pages/ParentBooking/ParentBookingContainer.js (hunk 3 above)
Comment (JesusNavarrete215) [reply src/components/pages/ParentBooking/ParentBookingContainer.js:66]: Returning the data in the other tabs will need to be a separate PR.

File: src/index.js [hunk 9]
```diff
@@ -67,7 +67,7 @@ function App() {
 
   return (
     <Security {...config} onAuthRequired={authHandler}>
-      {/* <NavBar /> */}
+      <NavBar />
```
Comment (RoseGabriely) [inline src/index.js:70]: Perfect!

File: src/components/pages/ParentBooking/ParentBookingCard.js [hunk 10]
```diff
@@ -4,21 +4,19 @@ import { dateConverter } from '../../common/dateHelpers';
...
-//   })
-//   .then(function (response) {
-//     console.log(response);
-//   })
-//   .catch(function (error) {
-//     console.log(error);
-//   });
-// }
+  axios
+    .post('/:id/enrollments')
```
Comment (RoseGabriely) [inline src/components/pages/ParentBooking/ParentBookingCard.js:12]: id needs to be dynamic here, also if this PR is ready to merge I'd probably get rid of this and do it in a different PR as it is extremely incomplete.

**Review by RoseGabriely — APPROVED** [review]
Good job implementing feedback. Last thing you'll need to do is decide if you want to implement the endpoints in this PR or start a new one for that.

File: src/components/pages/ParentBooking/ParentBookingCard.js (hunk 10 above)
Comment (JesusNavarrete215) [reply src/components/pages/ParentBooking/ParentBookingCard.js:12]: I will remove it and keep notes on it. Should look clean.

**Review by MarkTrepanier — CHANGES_REQUESTED** [review]
Amazing work and explanation of the changes and effects. Right now there is a merge conflict due to changes in the global.less. Would you mind going over that with whoever appropriate?

**Review by jonahjtr — APPROVED** [review]
theres a button  on line 57 in parent booking card that isn't finished with an onclick or anything, not sure if this is just something for later or just an accidental miss?

**Review by JesusNavarrete215 — COMMENTED** [review]
everything looks good on my end

**Review by RoseGabriely — COMMENTED** [review]
Looks great. All that's left is addressing Mark's request to resolve the merge conflicts, and address  Jaehun's comment on the extra axios import. Good work on this PR!
